src/Components/ListArticles.jsx

```diff
@@ -0,0 +1,52 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Button, Card, ListGroup} from "react-bootstrap";
+
+export const ListArticles = () => {
+  const [articles, setArticles] = useState([]);
+

```

Comment:
Should add some functionality to visually communicate to a user if the content is loading or not (right now it's just a blank page with a title while loading)